SparkleDotNET/SUAppcastItem.cs

```diff
@@ -39,6 +39,7 @@ public class SUAppcastItem : IComparable<SUAppcastItem> {
                 ItemDescription = (string)dict.ValueForKey("description");
                 FileURL = new Uri((string)enclosure.ValueForKey("url"));
                 DSASignature = (string)enclosure.ValueForKey("sparkle:dsaSignature");
+                RSASignature = (string)enclosure.ValueForKey("sparkle:rsaSignature");

```

Comment:
@prozacchiwawa - I extracted the public key from the certificate on Windows:
http://screencast.com/t/5nAZYnSRG6I

I think the "thumbprint" section is the hash of the certificate though, not for the binary.